package.json

```diff
@@ -427,7 +427,8 @@
     "history@^4.9.0": "patch:history@npm%3A4.10.1#./.yarn/patches/history-npm-4.10.1-ee217563ae.patch",
     "redux": "^5.0.0",
     "@storybook/blocks@npm:8.1.6": "patch:@storybook/blocks@npm%3A8.1.6#~/.yarn/patches/@storybook-blocks-npm-8.1.6-892f57a6d7.patch",
-    "react-grid-layout": "patch:react-grid-layout@npm%3A1.4.4#~/.yarn/patches/react-grid-layout-npm-1.4.4-4024c5395b.patch"
+    "react-grid-layout": "patch:react-grid-layout@npm%3A1.4.4#~/.yarn/patches/react-grid-layout-npm-1.4.4-4024c5395b.patch",
+    "@grafana/eslint-config": "portal:/Users/jackwestbrook/dev/grafana/eslint-config-grafana"
```

Comment:
(If you're testing this locally, don't forget to delete this before running `yarn link ...`)

Reply:
It was more to clean the lockfile until the tsconfig PR is merged. I'm hoping CI won't let us merge this like this. 🤞😄